### PR TITLE
deploy to ibm cloud

### PR DESCRIPTION
**Pull request by vabarbosa**
#24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,19 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+[![Deploy to IBM Cloud](https://metrics-tracker.mybluemix.net/stats/5c5df69e10058d49cdc1f4d2fc63ce31/button.svg)](https://bluemix.net/deploy?repository=https://github.com/ibm-watson-data-lab/shopping-list-vanillajs-pouchdb)
+
+1. Press the above ``Deploy to IBM Cloud`` button and then click on ``Deploy``.
+
+1. In Toolchains, click on Delivery Pipeline to watch while the app is deployed. Once deployed, the app can be viewed by clicking `View app`.
+
+1. To see the app and services created and configured for this code pattern, use the IBM Cloud dashboard. The app is named `shopping-list-vanillajs-pouchdb` with a unique suffix. The following services are created and easily identified by the `sljsp-` prefix:
+    * sljsp-CloudantNoSQLDB
+
 ## Run locally
```
[the remark this answers is not in the document]
Comment (vabarbosa) [reply README.md]: @bradley-holt instructions have been updated accordingly: https://github.com/ibm-watson-data-lab/shopping-list-vanillajs-pouchdb/pull/33/commits/e95d0468f3de256831db4f21a2f4b19b276f5100

**Comment by bradley-holt** [comment]
LGTM

File: manifest.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+declared-services:
```
Comment (ptitzler) [inline manifest.yml]: The file should start with `---`: or else the declared services won't be recognized:

```
---
declared-services:
  sljsp-cloudantNoSQLDB:
    label: cloudantNoSQLDB
    plan: Lite
applications:
...
```

**Comment by bradley-holt** [comment]
This looks good to me. What do you think, @ptitzler?

**Comment by ptitzler** [comment]
I can't get the deployment to Cloud button to work. The delivery pipeline does not provision the declared services and the operation therefore fails. I checked `manifest.yml` but can't find anything that's wrong.

**Comment by ptitzler** [comment]
The displayed deployment stats are not correct but that is expected. We'll take care of that in https://github.com/ibm-watson-data-lab/shopping-list-vanillajs-pouchdb/issues/21

**Comment by ptitzler** [comment]
Verified that deployment to Cloud works using the D2C button.

**Comment by bradley-holt** [comment]
@ptitzler Does that mean that this is working now? Can I go ahead and merge this?

**Pull request merged**